t4.cpp

```diff
@@ -1,4 +1,6 @@
 namespace {
 intt;
 int x = 1.1;
```

Comment:
implicit conversion from 'double' to 'int' changes value from 1.1 to 1 [clang-diagnostic-literal-conversion]